### PR TITLE
resolved the issues being faced

### PR DESCRIPTION
**Pull request by loma373**
Hi, I resolved the issues that were being faced. Lets merge to avoid others getting this issue.

### CODE REVIEW

**Comment by likerRr** [comment]
hi! thanks for participating

buy what was the issue about?

File: public/resume.html
Comment (likerRr) [inline public/resume.html:1]: why did you delete html and text resumes?

File: src/Speakable.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var util = require('util');
```
Comment (likerRr) [inline src/Speakable.js:1]: what was the purpose of switching from print to console?

File: src/dictionary.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ module.exports = {
     skills: ['skills', 'Skills & Expertise', 'technology', 'technologies'],
     languages: ['languages'],
     courses: ['courses'],
-    projects: ['projects'],
+    projects: ['projects','personal projects'],
```
Comment (likerRr) [inline src/dictionary.js]: please, add a whitespace after the comma

File: src/libs/processing.js [hunk 3]
```diff
@@ -48,7 +48,8 @@ function cleanTextByRows(data) {
  * @param cbAfterExtract
  */
 function extractText(file, cbAfterExtract) {
-  textract(file, {preserveLineBreaks: true}, function(err, data) {
+  console.log(file)
```
Comment (likerRr) [inline src/libs/processing.js]: remove redundant console.log

File: src/libs/processing.js [hunk 4]
```diff
@@ -48,7 +48,8 @@ function cleanTextByRows(data) {
  * @param cbAfterExtract
  */
 function extractText(file, cbAfterExtract) {
-  textract(file, {preserveLineBreaks: true}, function(err, data) {
+  console.log(file)
+  textract.fromFileWithPath(path.resolve(file), {preserveLineBreaks: true}, function(err, data) {
```
Comment (likerRr) [inline src/libs/processing.js:51]: I think this line is the main point of the issue?

**Review by likerRr — CHANGES_REQUESTED** [review]

File: src/libs/processing.js (hunk 4 above)
Comment (loma373) [reply src/libs/processing.js:51]: yep. I just resolved the issue that was being faced with this line.

File: src/Speakable.js (hunk 1 above)
Comment (loma373) [reply src/Speakable.js:1]: print was working and the error was saying that its outdated.

File: src/dictionary.js (hunk 2 above)
Comment (loma373) [reply src/dictionary.js]: I'll make the necessary changes

**Comment by likerRr** [comment]
@loma373 thanks!
